inventoi.php

```diff
@@ -414,17 +420,31 @@
               }
             }
 
+            // Lasketaan paikan varastosaldo,
+            // jotta sit� voidaan verrata sy�tettyihin m��riin
+            // ja saadaan tarkistettua relatiivisen m��r�n oikeellisuus oikein
+            $query = "SELECT
```

Comment:
@hannahagberg Tuotepeikat tiedot haetaan ennestään jo rivillä 468, eli siirtämällä se query hieman ylöspäin niin ei tarvii kannsta hakea samoja tietoja moneen kertaan.

Tässä queryssä on myös hieman epäloogisesti sum()-funktio selectissä vaikka wheressä rajataan tuotepaikan tunnuksella joka on uniikki.